shared/src/iosMain/kotlin/com/ajay/seenu/expensetracker/DatabaseDriver.ios.kt

```diff
@@ -0,0 +1,10 @@
+package com.ajay.seenu.expensetracker
+
+import app.cash.sqldelight.db.SqlDriver
+import app.cash.sqldelight.driver.native.NativeSqliteDriver
+
+actual class DriverFactory {
+    actual fun createDriver(): SqlDriver {
+        return NativeSqliteDriver(ExpenseDatabase.Schema, "test.db")
```

Comment:
If we are repeating the DB name across all modules, then its better to move it to a constant na.

Reply:
contant added